### PR TITLE
hv: add one more guest VM to the industry scenario

### PR DESCRIPTION
**Pull request by lyan3**
This commmit added one more guest VM to the industry scenario, to
be aligned with the HLD for the industry usage.

Tracked-On: #3039
Signed-off-by: Yan, Like <like.yan@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by jsun26intel** [comment]
@lyan3 
This need changes in description of INDUSTRY Kconfig of  also.

File: hypervisor/scenarios/industry/vm_configurations.c [hunk 1]
```diff
@@ -64,5 +64,20 @@ struct acrn_vm_config vm_configs[CONFIG_MAX_VM_NUM] = {
 			.addr.port_base = INVALID_COM_BASE,
 		}
 
+	},
+	{
+		.type = POST_LAUNCHED_VM,
```
Comment (dongyaozu) [inline hypervisor/scenarios/industry/vm_configurations.c]: This is renamed to .load_order now.

Comment (lyan3) [reply hypervisor/scenarios/industry/vm_configurations.c]: Rebased.

**Comment by lyan3** [comment]
Thanks. Both VM number and VM type have been updated.

> @lyan3
> This need changes in description of INDUSTRY Kconfig of also.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by wenlingz** [comment]
OK to verify

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**